samples/nrf5340/netboot/pm.yml

```diff
@@ -0,0 +1,19 @@
+#include <autoconf.h>
+#include <devicetree_legacy_unfixed.h>
+
+b0n_image:
+#if CONFIG_NETBOOT_MIN_PARTITION_SIZE
+  size: 0x1600
```

Comment:
Add some comments here regarding the size of the min config and the granularity of the locking